examples/accesscontrol/sysptrace_fail.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  labels:
+    run: sysptrace-fail
+    app: test
+    test-network-function.com/generic: target
+    test-network-function.com/container: target
+  name: sysptrace-fail
+  namespace: tnf
+spec:
+  shareProcessNamespace: true
+  automountServiceAccountToken: false
+  securityContext:
+    runAsUser: 1000900000
```

Comment:
Where do you get these userid from? Could we have some comments here?

Reply:
just random from the net